app/styles/app.scss

```diff
@@ -1,5 +1,7 @@
 @import 'variables';
 @import 'ember-paper';
+@import "ember-paper-expansion-panel";
```

Comment:
Ugh I missed removing that. I toyed around with expansion panels in the select course dialog before sticking with tabs